### ISSUE TITLE
Favorites Group always showing all favorite items whether or not they're in your bag (checkbox setting does nothing)

### ISSUE THREAD

**Issue by serasval** [Auctionator/Auctionator#1448]
Please include as much of the following information as possible to help me fix the bug:

## Brief Description of Bug
Favorites Group on the Selling tab behaves as though "Always show favorite items, including when not in your bag" is checked no matter what state the setting is in. 
Edit: This started after an update some time in the last couple of weeks.

## Steps to Reproduce

As far as I can tell there aren't really any steps beyond having a favorites group.

## Loaded Addons

Advanced Interface Options
Arkinventory
Astral Keys
Auctionator (obviously)
AutoLootPlus
BartrubySummonPet
Battle Pet BreedID
Better Loot Message
BetterWardrobe
BigWigs
BtWLoadouts
BtWQuests
Can I Mog It?
CopyPasta2
CraftSim
DejaCharacterStats
Demodal
Details! Damage Meter
DialogKey DF
ElvUI
FixRaid
GTFO
Guild Roster Manager
Guilds of WoW
HandyNotes
HidingBar
ImprovedTalentLoadOuts
Kaliel's Tracker
LiteMount
LittleWigs
Macro Toolkit
Method Raid Tools
Misspelled
Mythic Dungeon Tools
NameplateSCT
Narcissus
No Mats; No Make
OmniCC
OmniCD
Pawn
Premade Groups Filter
Raider.IO Mythic Plus
RCLootCouncil
SimulationCraft
Talent Tree Tweaks
TL;DR Missions
TomTom
TradeSkillMaster
True Stat Values
Visage Mount
VocalRaidAssistant
VuhDo
WeakAuras
World Quest Tracker
WorldBossTimers

## Back Trace (LUA Error Output)

N/A

**Comment by plusmouse** [comment]
Thanks for the bug report. I've added a fix into the latest alpha build (available shortly from Curseforge)
